packages/plugins/RSS3/src/SNSAdaptor/components/DonationCard.tsx

```diff
@@ -1,103 +1,100 @@
+import type { HTMLProps } from 'react'
+import { RSS3_DEFAULT_IMAGE } from '../../constants'
 import { useI18N } from '../../locales'
 
 export interface DonationCardProps extends HTMLProps<HTMLDivElement> {
-    imageUrl: string
-    name: string
-    contribCount: number
-    contribDetails: Array<{
-        token: string
-        amount: string
-    }>
+    donation: RSS3BaseAPI.Donation
+    address: SocialAddress<NetworkPluginID>
+    onSelect: () => void
 }
 
 const useStyles = makeStyles()((theme) => ({
     card: {
         borderRadius: 8,
         display: 'flex',
         flexDirection: 'row',
-        backgroundColor: MaskColorVar.twitterBg,
-        padding: theme.spacing(1),
         flexGrow: 1,
         alignItems: 'stretch',
+        padding: 3,
+        cursor: 'pointer',
     },
     cover: {
         flexShrink: 1,
-        height: 90,
-        width: 90,
+        height: 126,
+        width: 126,
         borderRadius: 8,
         objectFit: 'cover',
     },
-    title: {
-        color: theme.palette.text.primary,
-        fontSize: 16,
+    date: {
+        color: theme.palette.maskColor.main,
+        fontSize: 14,
+        fontWeight: 400,
         whiteSpace: 'nowrap',
         overflow: 'hidden',
         textOverflow: 'ellipsis',
     },
     info: {
-        flexGrow: 1,
-        marginLeft: theme.spacing(1),
+        marginTop: 15,
+        marginLeft: '12px',
         fontSize: 16,
-        display: 'flex',
-        overflow: 'hidden',
-        flexDirection: 'column',
-        justifyContent: 'space-around',
-        fontFamily: '-apple-system,system-ui,sans-serif',
     },
     infoRow: {
-        whiteSpace: 'nowrap',
+        marginBottom: 8,
         overflow: 'hidden',
         textOverflow: 'ellipsis',
     },
-    infoLabel: {
-        color: theme.palette.text.primary,
+    activity: {
+        fontSize: 14,
+        fontWeight: 400,
+        fontColor: theme.palette.maskColor.main,
     },
-    infoValue: {
-        color: theme.palette.text.secondary,
+    fontColor: {
+        color: theme.palette.maskColor.primary,
     },
 }))
 
-export const DonationCard = ({
-    imageUrl,
-    name,
-    contribCount,
-    contribDetails,
-    className,
-    ...rest
-}: DonationCardProps) => {
+export const DonationCard = ({ donation, address, onSelect, className, ...rest }: DonationCardProps) => {
     const { classes } = useStyles()
     const t = useI18N()
+    const { value: domain } = useReverseAddress(address.networkSupporterPluginID, address.address)
+    const { Others } = useWeb3State(address.networkSupporterPluginID)
+    const reversedAddress =
+        !domain || !Others?.formatDomainName
+            ? Others?.formatAddress?.(address.address, 5) ?? address.address
+            : Others.formatDomainName(domain)
+
+    const date = donation.detail?.txs?.[0]
+        ? formatDateTime(new Date(Number(donation.detail?.txs?.[0]?.timeStamp) * 1000), 'MMM dd, yyyy')
```

Comment:
```ts
import fromUnixTime from 'date-fns/fromUnixTime'
```

Reply:
but how to format date as `MMM dd, yyyy`